Date/TimeZone.php

```diff
@@ -431,7 +430,7 @@ function getDefault()
      * @return   void
      * @access   public
      */
-    function setDefault($id)
+    static function setDefault($id)
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.